src/content/getting-started/cloud-provider-accounts/cluster-api/azure/index.md

```diff
@@ -0,0 +1,138 @@
+---
+linkTitle: Azure
+title: Prepare an Azure subscription to run Cluster API Giant Swarm cluster
+description: This guide will walk you through all necessary steps to set up an Azure subscription with appropriate IAM roles for operating Cluster API Giant Swarm clusters.
+menu:
+  main:
+    identifier: gettingstarted-cloudprovider-clusterapi-azure
+    parent: gettingstarted-cloudprovider-clusterapi
+weight: 20
+user_questions:
+  - How do I prepare my Azure subscription for Cluster API?
+  - What do I need to configure in Azure in order to run Cluster API Giant Swarm clusters?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-10-16
+---
+
+Currently, the Giant Swarm Cluster API for Azure implementation supports only running the Management and Workload Clusters in a single subscription.
+
+## Overview
+
+This document consists of the instructions for setting up Azure subscriptions that are needed to run Cluster API for Azure with Giant Swarm.
+
+## Prerequisites
+
+There are two main groups of requirements to be met before going over next steps:
+
+1. Azure subscription must be chosen from existing ones or created if needed in customer Azure account. For security reasons in terms of any sensitive data stored within customer accounts, we advise to use subscription only designated for Giant Swarm Azure resources management
+
+2. An individual has to have following permissions and tools working within the designated Azure subscription:
+
+* An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+## Configure subscription to allow access for Giant Swarm Support
```

Comment:
```suggestion
### Configure subscription to allow access to Giant Swarm Support
```
In line 84 you said that previous points are prerequisites, so they need another level